aat/src/aat/resources/features/F-083/S-359.td.json

```diff
@@ -1,119 +1,121 @@
-		"_extends_": "Common_200_Response",
-		"headers": {
-			"Content-Encoding": "[[ANYTHING_PRESENT]]",
-			"Content-Length": "[[ANYTHING_PRESENT]]",
-			"Content-Type": "[[ANYTHING_PRESENT]]",
-			"Strict-Transport-Security": "[[ANY_STRING_NULLABLE]]"
-		},
-		"body": {
-			"case_type_id": "FT_DateTimeFormats",
-			"fields": [
-				{
-					"__ordering__": "unordered",
-					"__elementId__": "order"
-				},
-				{
-					"case_field_id": "[LAST_STATE_MODIFIED_DATE]",
-					"case_field_element_path": null,
-					"label": "Last State Modified Date",
-					"order": 1,
-					"role": null,
-					"display_context_parameter": "#DATETIMEENTRY(yyyy)",
-					"show_condition": null
-				},
-				{
-					"case_field_id": "DateField",
-					"case_field_element_path": null,
-					"label": "Formatted Date",
-					"order": 2,
-					"role": null,
-					"display_context_parameter": "#DATETIMEENTRY(yyyy)",
-					"show_condition": null
-				},
-				{
-					"case_field_id": "DateTimeField",
-					"case_field_element_path": null,
-					"label": "Formatted DateTime",
-					"order": 3,
-					"role": null,
-					"display_context_parameter": "#DATETIMEENTRY(yyyy-MM-dd'T'HH:mm:ss)",
-					"show_condition": null
-				},
-				{
-					"case_field_id": "StandardDate",
-					"case_field_element_path": null,
-					"label": "Standard Date",
-					"order": 4,
-					"role": null,
-					"display_context_parameter": null,
-					"show_condition": null
-				},
-				{
-					"case_field_id": "StandardDateTime",
-					"case_field_element_path": null,
-					"label": "Standard DateTime",
-					"order": 5,
-					"role": null,
-					"display_context_parameter": null,
-					"show_condition": null
-				},
-				{
-					"case_field_id": "CollectionDate",
-					"case_field_element_path": null,
-					"label": "Date Collection",
-					"order": 6,
-					"role": null,
-					"display_context_parameter": "#DATETIMEENTRY(yy-MM-dd)",
-					"show_condition": null
-				},
-				{
-					"case_field_id": "CollectionDateTime",
-					"case_field_element_path": null,
-					"label": "DateTime Collection",
-					"order": 7,
-					"role": null,
-					"display_context_parameter": "#DATETIMEENTRY(yyyy-MM-dd)",
-					"show_condition": null
-				},
-				{
-					"case_field_id": "CollectionComplexDateTime",
-					"case_field_element_path": null,
-					"label": "Collection Complex DateTime",
-					"order": 8,
-					"role": null,
-					"display_context_parameter": null,
-					"show_condition": null
-				},
-				{
-					"case_field_id": "ComplexDateTime",
-					"case_field_element_path": "DateField",
-					"label": "Complex Date",
-					"order": 9,
-					"role": null,
-					"display_context_parameter": null,
-					"show_condition": null
-				},
-				{
-					"case_field_id": "ComplexDateTime",
-					"case_field_element_path": "DateTimeField",
-					"label": "Complex DateTime",
-					"order": 10,
-					"role": null,
-					"display_context_parameter": null,
-					"show_condition": null
-				}
-			]
-		}
-	}
+  "_guid_": "S-359",
```

Comment:
auto format with tabs not applied correctly to this and some other files.

Reply:
updated